src/Admin/Vue.php

```diff
@@ -377,14 +377,14 @@ public function thresholds_option($max, $min) {
 					"name" => "oney_min_amounts",
 					"value" => $min,
 					"placeholder" => $min,
-					"default" => get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_min']
+					"default" => !empty(get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_min']) ? get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_min'] : 100
 				],
 				"inter" => __( 'and', 'payplug' ),
 				"max_amount" => [
 					"name" => "oney_max_amounts",
 					"value" => $max,
 					"placeholder" => $max,
-					"default" => get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_max']
+					"default" => !empty(get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_max']) ? get_option( 'woocommerce_payplug_settings', [] )['oney_thresholds_default_max'] : 3000
```

Comment:
we have the thresholds coming as parameters, it should be using for the fail safe instead of static value